src/Java/ImageData/ImageNormalize.java

```diff
@@ -0,0 +1,91 @@
+import vtk.vtkNativeLibrary;
+import vtk.vtkRenderWindow;
+import vtk.vtkRenderWindowInteractor;
+import vtk.vtkRenderer;
+import vtk.vtkImageSinusoidSource;
+import vtk.vtkImageNormalize;
+import vtk.vtkImageCast;
+import vtk.vtkImageActor;
+
+public class ImageNormalize 
+{
+  // -----------------------------------------------------------------
+  // Load VTK library and print which library was not properly loaded
+  static 
+  {
+    if (!vtkNativeLibrary.LoadAllNativeLibraries()) 
+    {
+      for (vtkNativeLibrary lib : vtkNativeLibrary.values()) 
+      {
+        if (!lib.IsLoaded()) 
+        {
+          System.out.println(lib.GetLibraryName() + " not loaded");
+        }
+      }
+    }
+    vtkNativeLibrary.DisableOutputWindow(null);
+  }
+  // -----------------------------------------------------------------
+
+  public static void main(String args[]) 
+  {
+    //  Create an image
+    vtkImageSinusoidSource source = new vtkImageSinusoidSource();
+    source.Update();
+
+    vtkImageNormalize normalizeFilter = new vtkImageNormalize();
+
+    normalizeFilter.SetInputConnection(source.GetOutputPort());
+    normalizeFilter.Update();
+    
+    vtkImageCast inputCastFilter = new vtkImageCast();
+    inputCastFilter.SetInputConnection(source.GetOutputPort());
+    inputCastFilter.SetOutputScalarTypeToUnsignedChar();
+    inputCastFilter.Update();
+
+    vtkImageCast normalizeCastFilter = new vtkImageCast();
+    normalizeCastFilter.SetInputConnection(normalizeFilter.GetOutputPort());
+    normalizeCastFilter.SetOutputScalarTypeToUnsignedChar();
+    normalizeCastFilter.Update();
+    
+    //  Create actors
+    vtkImageActor inputActor = new vtkImageActor();
+    inputActor.GetMapper().SetInputConnection(inputCastFilter.GetOutputPort());
+
+    vtkImageActor normalizedActor = new vtkImageActor();
+    normalizedActor.GetMapper().SetInputConnection(normalizeCastFilter.GetOutputPort());
+    
+    //  Create the render window and interactor.
+    vtkRenderWindow renWin = new vtkRenderWindow();
+    vtkRenderWindowInteractor iren = new vtkRenderWindowInteractor();
+    iren.SetRenderWindow(renWin);
+    
+    //  Define viewport ranges
+    // (xmin, ymin, xmax, ymax)
+    double leftViewport[] = new double[] {0.0, 0.0, 0.5, 1.0};
+    double rightViewport[] = new double[] {0.5, 0.0, 1.0, 1.0};
+    
+    //  Setup both renderers
+    vtkRenderer leftRenderer = new vtkRenderer();
+    renWin.AddRenderer(leftRenderer);
+    leftRenderer.SetViewport(leftViewport);
+    leftRenderer.SetBackground(.6, .5, .4);
+
```

Comment:
Once again, use `vtkNamedColors`, it is bad practise to do what is done here, and the guidelines also emphasise using `vtkNamedColors`. The old C++ examples were made before `vtkNamedColors` existed, and are gradually being updated. The simplest way in this case is:
```java
double left_bkg[] = new double[]{0.6, 0.5, 0.4, 1.0};
double right_bkg[] = new double[]{0.4, 0.5, 0.6, 1.0};
```
If you do this you won't have to update the image.

Reply:
Changes made as suggested.

Reply:
As i'll try to keep things as same as C++ program, I tend to do the as it is.
Now on i'll do appropriate updates required. 
Please review now.

Reply:
Yes thanks. We're trying to make the examples  look as modern as possible. Some of the  C++ predates the introduction of vtkNamedColors - it was specifically written so users could simply name colors from a predefined pallete or create their own.